docs/spicedb.md

```diff
@@ -509,7 +509,7 @@ spicedb serve [flags]
       --http-enabled                                                                    enable http proxy server
       --http-tls-cert-path string                                                       local path to the TLS certificate used to serve proxy
       --http-tls-key-path string                                                        local path to the TLS key used to serve proxy
-      --lookup-resources-chunk-cache-enabled                                            enable caching (default true)
+      --lookup-resources-chunk-cache-enabled                                            enable caching of LookupResources3 chunks (default true)
```

Comment:
why is this LR3 only? Why do the other `--lookup-resources-*` not need such clarification?